src/gui/optionsdlg.ui

```diff
@@ -2362,10 +2362,124 @@
                 </spacer>
                </item>
                <item row="3" column="0" colspan="3">
-                <widget class="QCheckBox" name="checkIgnoreSlowTorrentsForQueueing">
-                 <property name="text">
+                <widget class="QGroupBox" name="checkIgnoreSlowTorrentsForQueueing">
+                 <property name="title">
                   <string>Do not count slow torrents in these limits</string>
                  </property>
+                 <property name="checkable">
+                  <bool>true</bool>
+                 </property>
+                 <property name="checked">
+                  <bool>false</bool>
+                 </property>
+                 <layout class="QGridLayout" name="gridLayout_8">
+                  <item row="2" column="3">
+                   <spacer name="horizontalSpacer_21">
+                    <property name="orientation">
+                     <enum>Qt::Horizontal</enum>
+                    </property>
+                    <property name="sizeHint" stdset="0">
+                     <size>
+                      <width>40</width>
+                      <height>20</height>
+                     </size>
+                    </property>
+                   </spacer>
+                  </item>
+                  <item row="0" column="1">
+                   <widget class="QSpinBox" name="spinDownloadRateForSlowTorrents">
+                    <property name="suffix">
+                     <string> KiB/s</string>
+                    </property>
+                    <property name="maximum">
+                     <number>2000000</number>
+                    </property>
+                    <property name="value">
+                     <number>2</number>
+                    </property>
+                   </widget>
+                  </item>
+                  <item row="1" column="1">
+                   <widget class="QSpinBox" name="spinUploadRateForSlowTorrents">
+                    <property name="suffix">
+                     <string> KiB/s</string>
```

Comment:
I just notice this style is different to rate limits in speed tab.
Which one is better? And we should try unify it.

Reply:
But which one? Behavior -> Log file uses this format.

Reply:
>But which one? 

I think using qspinbox suffix is better (1 less label widget), so other places should convert.
You may handle this one if you're interested.

Reply:
I will.